src/components/RecipesList.js

```diff
@@ -58,6 +61,11 @@ class RecipesList extends React.Component {
           })
           }
         </ul>
+
```

Comment:
Just for my own curiosity are there supposed to be blank lines around this div in terms of styling?

Reply:
I left over from how when I was working as it helps me see what i am working on. Will remove.

Reply:
👍